pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --cov . --pep8 --pylint --cov-report term --cov-report html --ds=micromasters.settings --reuse-db
+addopts = --cov . --pep8 --pylint --cov-report term --cov-report html --ds=micromasters.settings --reuse-db --create-db
```

Comment:
This is being added so the db gets created and migrations get run before the first test case runs. This only happens at the beginning of the session